data/os/RedHat.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-nfsclient::service_name:      'auth-rpcgss-module.service'
+nfsclient::service_name:      'gssproxy'
```

Comment:
Check, `$service_name` is only used when `$nfs_config_method` is set to `service` which isn't the default for RH6/7.

Reply:
Yeah.  AFIAK RHEL6 and RHEL7 uses sysconfig. RHEL8 and RHEL9 uses this service instead. On RHEL8 auth-rpcgss-module is used while on RHEL9 uses gssproxy (and it works correctly). However it's interesting because auth-rpcgss-module exists on RHEL9 as well, however starting gssproxy is enough. I'm not sure if maybe both are supposed to be started.

auth-rpcgss-module appears to be loading a kernel module. gssproxy starts a service. Maybe the module loading happens automatically when gssproxy is started.